Frontend/android-native/app/src/user/nammaYatri/res/values/ic_launcher_background.xml

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="ic_launcher_background">#2C2F3A</color>
+    <color name="ic_launcher_background">#14141C</color>
```

Comment:
@ashishsingh101 please change the color to #13131A from #14141C

Reply:
done